2018/README.md

```diff
@@ -0,0 +1,111 @@
+# Notes for 2018
+
+The templates here are intended to be 1:1 scale diagram of all the areas used in SUSU, they are believed to be within 5% accurate.
+Locations highlighted in red on one of the venue related layers require measurements to be taken to assure their size.
+
+#### Requires:
+- \> python 3.4
```

Comment:
The `generate_svg.py` script uses f-strings, which are in Python 3.6+ only. Should the f-strings be removed, or is this requirement wrong?

Reply:
This is an error, it should be 3.6